cmd/keytransparency-sequencer/server.go

```diff
@@ -25,12 +25,13 @@ import (
 	"google.golang.org/grpc/credentials"
 )
 
-func startHTTPServer(grpcServer *grpc.Server, addr string,
-	services ...serverutil.RegisterServiceFromEndpoint) *http.Server {
-	// Wire up gRPC and HTTP servers.
-	tcreds, err := credentials.NewClientTLSFromFile(*certFile, "")
-	if err != nil {
-		glog.Exitf("Failed opening cert file %v: %v", *certFile, err)
+func serveHTTPMetric(port string) {
+	metricMux := http.NewServeMux()
+	metricMux.Handle("/metrics", promhttp.Handler())
+
+	glog.Infof("Hosting metrics on %v", port)
+	if err := http.ListenAndServe(port, metricMux); err != nil {
+		glog.Fatalf("ListenAndServeTLS(%v): %v", *metricsAddr, err)
 	}
 	gwmux, err := serverutil.GrpcGatewayMux(addr, tcreds, services...)
```

Comment:
When is this meant to run? `ListenAndServe` above returns only when the server is stopped.

Reply:
It's meant to run in it's own go routine like so:
```go
go serveHTTPMetric(addr)
```